src/main/java/org/assertj/core/internal/Paths.java

```diff
@@ -496,4 +499,22 @@ private static void assertExpectedEndPathIsNotNull(final Path end) {
     requireNonNull(end, "the expected end path should not be null");
   }
 
+  public void assertIsEmptyPath(AssertionInfo info, Path actual) {
+    assertIsRegularFile(info, actual);
+    try {
+      if (size(actual) == 0) return;
+    } catch (IOException e) {
+
+    }
+    throw failures.failure(info, shouldBeEmpty(actual));
+  }
+  public void assertIsNotEmptyPath(AssertionInfo info, Path actual){
```

Comment:
Let's add a line between the methods. I would also rename this one to avoid confusion with directories:
```suggestion

  public void assertIsNotEmptyFile(AssertionInfo info, Path actual) {
```

Reply:
Perfect! Thanks for the feedback. I'm work on it.